apps/mull-api/src/app/auth/auth.guard.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { createMock } from '@golevelup/nestjs-testing';
```

Comment:
Assumed since I was testing the function I would get coverage but I found this [github issue](https://github.com/nestjs/nest/issues/1020) and applied it and it gives us our line coverage while testing the function too.